tests/platform_tests/test_reload_config.py

```diff
@@ -141,6 +141,11 @@ def test_reload_configuration_checks(duthosts, enum_rand_one_per_hwsku_hostname,
     @summary: This test case is to test various system checks in config reload
     """
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    hwsku = duthost.facts["hwsku"]
+
+    config_reload_timeout = 120
+    if hwsku in ["Nokia-M0-7215", "Nokia-7215"]:
+        config_reload_timeout = 180
```

Comment:
Is it the day1 failure? or the failure was introduced by the execute_config_reload_cmd change?

Reply:
I checked the test history, it should not be introduced by testcase change.
I can see this testcase always pass before fips is enabled in image. So it looks like fips introduce more time to `config load` and test start to fail.